packages/daisyui/babel.config.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "../../babel.config.json"
+}
```

Adding blank line at end to remove the red circled dash
```suggestion
}

```

---

Fighting my linter. I'll fix.